test

```diff
@@ -89,15 +98,35 @@ function grpcproxy_pass {
 }
 
 function release_pass {
+	rm -f ./bin/etcd-last-release
 	# to grab latest patch release; bump this up for every minor release
 	UPGRADE_VER=$(git tag -l --sort=-version:refname "v3.0.*" | head -1)
 	if [ -n "$MANUAL_VER" ]; then
 		# in case, we need to test against different version
 		UPGRADE_VER=$MANUAL_VER
 	fi
-	echo "Downloading" etcd $UPGRADE_VER
-	curl -L https://github.com/coreos/etcd/releases/download/$UPGRADE_VER/etcd-$UPGRADE_VER-linux-amd64.tar.gz -o /tmp/etcd-$UPGRADE_VER-linux-amd64.tar.gz
-	tar xzvf /tmp/etcd-$UPGRADE_VER-linux-amd64.tar.gz -C /tmp/ --strip-components=1
+
+	local file="etcd-$UPGRADE_VER-linux-$MACHINE_ARCH.tar.gz"
+	echo "Downloading $file"
+
+	set +e
+	curl --fail --progress-bar -L https://github.com/coreos/etcd/releases/download/$UPGRADE_VER/$file -o /tmp/$file 2>&1
+	local result=$?
+	set -e
+	case $result in
+		0)	;;
+		22)
+			echo -e "\n$file not found"
+			return
+			;;
+		*)
+			echo -e "\nDownloading $file failed"
+			exit 255
+			;;
+	esac
+
+	tar xzvf /tmp/$file -C /tmp/ --strip-components=1
+	mkdir -p ./bin
```

Comment:
As I mentioned in the commit message, if you run ```PASSES="release" ./test```, the test will fail. This is because there is no bin directory in the etcd repo.